workflow/defaults/settings.yaml

```diff
@@ -58,6 +58,8 @@ default_settings: &default_settings
       do: .true.
     - otherwise: .false.
 
+  ic_source: coupled_ic  # choices: coupled_ic, fv3_ic; default: coupled_ic
```

Comment:
Can we add more of an explanation/definition of the variable here?  It seems obvious now, but adding information that it determines which script is used for ICs would be helpful to those who didn't follow this conversation I think.

Reply:
Your suggestion is taken